R/checkbox.R

```diff
@@ -197,9 +197,9 @@ multiple_checkbox <- function(input_id, label, choices, choices_value = choices,
 update_multiple_checkbox <- function(session = getDefaultReactiveDomain(),
                                      input_id, choices = NULL, choices_value = choices,
                                      selected = NULL, label = NULL) {
-  if (!is.null(selected)) value <- jsonlite::toJSON(selected) else value <- NULL
+  if (!is.null(selected)) value <- jsonlite::toJSON(gsub( "'", '"', selected)) else value <- NULL
```

Comment:
Let's remove the whitespace ;) 
```suggestion
  if (!is.null(selected)) value <- jsonlite::toJSON(gsub("'", '"', selected)) else value <- NULL
```